src/scss/pages/_degree-search.scss

```diff
@@ -34,3 +34,31 @@
     margin-top: 6px;
   }
 }
+
+#degree-search-query {
+  background-image: str-replace(url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 89 89'%3E%3Cpath fill='#{$gray-light}' d='M66.4,60.2 C71.5,53.8 74.3,45.7 74.3,37.5 C74.3,17.3 57.8,0.8 37.5,0.8 C17.2,0.8 0.8,17.3 0.8,37.5 C0.8,57.7 17.3,74.2 37.5,74.2 C45.6,74.2 53.7,71.4 60.2,66.3 L82.3,88.4 L88.6,82.1 L66.4,60.2 Z M37.5,65.5 C22.2,65.5 9.7,53 9.7,37.7 C9.7,22.4 22.2,9.9 37.5,9.9 C52.8,9.9 65.3,22.4 65.3,37.7 C65.3,53 52.8,65.5 37.5,65.5 Z'/%3E%3C/svg%3E"), "#", "%23");
```

Comment:
What is this SVG? Is this an icon that was not available in font awesome?

Reply:
It's just an SVG that's rendered directly for the background image. I used the same method as for how the form validation icons are inputted into form inputs fields with Athena: 
https://github.com/UCF/Athena-Framework/blob/master/src/scss/_variables.scss#L862
I was actually going to input this into Athena as well (referencing https://github.com/UCF/Athena-Framework/issues/41), so once that's in and this theme is using the most recent version of Athena, we will be able to remove this style and just insert the class onto that input field.

Reply:
OK, I like the idea I just I didn't realize we were going to start using SVGs.